src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingWorkspaceWrapper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Remote;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal readonly struct UnitTestingWorkspaceWrapper
```

Comment:
😕 Workspace is a public type. Why is this needed?

💡 Ah, you should be able to define a `TryGetRemoteHostClientAsync` extension method in an extension class `UnitTestingWorkspaceExtensions`, rather than wrapping the entire class.